impeller/aiks/aiks_unittests.cc

```diff
@@ -3492,6 +3493,19 @@ TEST_P(AiksTest, CanCanvasDrawPictureWithAdvancedBlend) {
   ASSERT_TRUE(OpenPlaygroundHere(canvas.EndRecordingAsPicture()));
 }
 
+TEST_P(AiksTest, CanDrawMultiContourConvexPath) {
+  PathBuilder builder = {};
+  for (auto i = 0; i < 10; i++) {
+    builder.AddCircle(Point(100 + 10 * i, 100 + 10 * i), 100);
```

Comment:
nit: Interlace triangles here too (so the appended contours will be Circle, Triangle, Circle, Triangle, etc.) that way we know we're testing the `a == b` branch for `TessellateConvex`, which will only trigger for contours with an odd number of points.

Reply:
Ahh good idea.